macros/edr/dbt_artifacts/upload_dbt_artifacts.sql

```diff
@@ -26,6 +26,10 @@
         {% do elementary.file_log('[{}] Artifacts already ran.'.format(artifacts_model)) %}
       {% endif %}
     {% endfor %}
+    {% if elementary.get_dbt_columns_materialization() != "view" %}
+      {% do adapter.commit() %}
```

Comment:
Is this needed? There's `should_commit=true` for the uploads.

Reply:
but we need to commit before running the `dbt_columns` query, as it queries the other artifacts

Reply:
Yes, but I'm saying that the there are no pending transactions at this point because of the [`should_commit=True`](https://github.com/elementary-data/dbt-data-reliability/blob/b618ea2ef9f689d31ae55c1c41135dbd0d2be505/macros/edr/dbt_artifacts/upload_dbt_artifacts.sql#L20) and each model commits when it finishes executing.